src/replication.thrift

```diff
@@ -478,6 +478,21 @@ struct query_disk_info_response
     4:list<disk_info> disk_infos;
 }
 
+// This request is sent from clent to replica_server.
+struct migrate_replica_request 
+{
+    1:dsn.rpc_address node;
+    2:dsn.gpid pid
+    3:string origin_disk;
+    4:string target_disk;
+}
+
+// This response is recieved replica_server.
+struct migrate_replica_response 
+{
+   1:dsn.error_code err;
```

Comment:
```suggestion
   1:dsn.error_code err;
   2:optional string hint;
```

Provide hints rather than prints error logs on server, so that clients won't have to check out the server log.